messages/teleporter/message_manager.go

```diff
@@ -186,8 +186,7 @@ func (m *messageManager) SendMessage(signedMessage *warp.Message, destinationBlo
 		return fmt.Errorf("relayer not configured to deliver to destination. DestinationBlockchainID=%s", destinationBlockchainID)
 	}
 
-	teleporterMessenger := m.getTeleporterMessenger(destinationClient)
-	teleporterMessageID, err := m.calculateMessageID(teleporterMessenger, signedMessage.SourceChainID, destinationBlockchainID, teleporterMessage.MessageNonce)
+	teleporterMessageID, err := teleporterutils.CalculateMessageID(m.protocolAddress, signedMessage.SourceChainID, destinationBlockchainID, teleporterMessage.MessageNonce)
```

Comment:
I previously decided to leave the log in `CalculateMessageID`, can we add logs for handling these errors with the chagned function

Reply:
I've wrapped the error here so it's clear where the error is coming from. We log any errors outside of this function, so they will get logged. I'd like to talk about our error logging in this repo in standup today if there's time.